src/test/scala/special/sigma/SigmaDslTest.scala

```diff
@@ -338,6 +506,13 @@ class SigmaDslTest extends PropSpec
      |}""".stripMargin)
   }
 
+  property("getVar equivalence") {
+    val eq = checkEq(func[Int,Int]("{ (x: Int) => getVar[Int](2).get }", 2.toByte -> IntConstant(10))) { x =>
+      10
```

Comment:
@gostkin this method should test equivalence between Scala and Script, when the text is IDENTICAL